Added explanation for search command ACLs in RedisSearch quick start documentation (#3146)
RediSearch/RediSearch: docs/docs/Quick_Start.md

`FT.SEARCH` expects valid UTF-8 or ASCII as input. The engine cannot handle wide character unicode.
{{% /alert %}}

{{% alert title="Note" color="info" %}}

> When configuring [ACLs](/docs/manual/security/acl/), search-related commands such as `FT.SEARCH` consider the index name to be the "key". Therefore, if key patterns are used in an ACL rule (dis)allowing search commands, a pattern matching the index name should be included for needed commands. 

Reviewer: ```suggestion
When configuring [ACLs](/docs/management/security/acl/), search-related commands such as `FT.SEARCH` consider the index name to be the "key". Therefore, if you use key patterns in an ACL rule allowing or disallowing search commands, include a pattern matching the index name for the specified commands. 
```

> For example, to enable use for searching the index `myIdx`, the ACL rule should include a pattern for either this command, or a category which includes it. Such ACL rules could be `+ft.search ~myIdx`, `+ft.search ~myId*`, `+@all ~myIdx` etc.

Reviewer: ```suggestion
For example, to enable search on an index `myIdx`, the ACL rule should include a pattern for either 'FT.SEARCH` or a category that includes it. Such ACL rules could be `+ft.search ~myIdx`, `+ft.search ~myId*`, `+@all ~myIdx`, and so on.
```
